templates/splash.html

```diff
@@ -408,11 +461,20 @@
     justify-content: space-between;
     align-items: flex-end;
   }
+  #now-playing {
+    padding: 10px 20px 10px 20px;
+    border-bottom-left-radius: 20px;  
+  }
   #qr-code {
     max-width: 25%;
+    border-top-right-radius: 20px; /* Rounded corner */
+    text-align: center; /* Center QR code */   
```

Comment:
I'd prefer for this to remain right-aligned so it's less likely to cover karaoke video text.

Reply:
can do, not a biggie. It looked a little awkward when I had a long url there... I have one instance spun up on my personal server, and I know it would look odd to have the QR code left aligned.... buuuut even centered, the long URL + the QR doesn't look the greatest graphically either.

Been pondering a way to make it look more visually appealing... or maybe make the url text disappear altogether once a certain character count is reached.

Reply:
Yeah, I agree its not great either way, maybe lowering the text size of the url would make it look more symmetrical. Or it might be worth adding a command line option to hide the url, but show the QR code, for minimalists. If that were the case a corner-aligned QR code would look pretty clean. But that's another ticket

Reply:
![image](https://github.com/vicwomg/pikaraoke/assets/72324013/01b62809-87e2-4634-996f-b9fbab265b0c)

This might be worth adding as a tick box option in the admin area.